src/pam-ssh-add/pam-ssh-add.c

```diff
@@ -149,9 +149,10 @@ message_handler (int level,
   va_end (va);
 
   if (res > 0)
```

Comment:
A comment about coding style:  Maybe it's slightly better to use

 if (res >= 0)

An empty message also needs to be allocated due to the ending '\0' in the string.

(I guess an empty message could not happen so this is not important)

Reply:
Right, I noticed that, but initially discarded the "empty message" edge case as that doesn't seem to happen in practice. But anyway, I fixed it properly.